sdk/anomalydetector/ai-anomaly-detector/package.json

```
@@ -65,13 +65,13 @@
     "@azure/core-http": "^1.2.0",
     "@azure/core-auth": "^1.2.0",
     "@azure/logger": "^1.0.0",
-    "@opentelemetry/api": "^0.10.2",
+    "@opentelemetry/api": "^1.0.0-rc.0",
```

Pre-release versions should be pinned instead of floated, since breaking changes are allowed across pre-release versions.

```suggestion
    "@opentelemetry/api": "1.0.0-rc.0",
```

---

Makes sense to me.

---

Done.